File: webservices/src/main/java/fieg/modulos/unidade/dto/UnidadeDTO.java
package fieg.modulos.unidade.dto;

import fieg.modulos.entidade.enums.Entidade;
import lombok.Data;

@Data
public class UnidadeDTO {
    private Integer id;
    private Integer ano;
    private String codigo;
    private String nome;
    private String descricaoUnidade;
    private Entidade entidade;
    private String filialERP;
    private String centroCustoErp;
    private Integer idLocal;
    private String cidade;
    private String uf;
}

File: web/src/models/DTOs/Outros/UnidadeDTO.ts
import { EntidadeEnum } from "./EntidadeDTO";

export interface UnidadeDTO {
  id: number;
  idLocal: number;
  codigo: number;
  filialERP: string;
  centroCustoErp: string;
  descricaoUnidade: string;
  entidade: EntidadeEnum;
  nome: string;
  cidade: string;
  uf: string;
}
